.templates/php/default.mk

```diff
@@ -33,20 +48,21 @@ post-release:
 .PHONY: post-release
```

Comment:
I guess here we should revert what has been done by `remove-local-repository`

Reply:
ok, something like `restore-local-repository`